coalib/misc/Caching.py

```diff
@@ -0,0 +1,142 @@
+import os
+import calendar
+import time
+import appdirs
+import pickle
```

Comment:
Yeah I think that's the way forward. Post a warning message, delete the 
corrupt file, re-cache.

All in all I like this very much :)


Am 15. April 2016 9:02:27 vorm. schrieb Adhityaa Chandrasekar 
<notifications@github.com>:

>> @@ -0,0 +1,142 @@
>> +import os
>> +import calendar
>> +import time
>> +import appdirs
>> +import pickle
>
> @fneu I think it will, at worst, corrupt the file. I didn't think about the 
> possibility of that happening.
>
> So I think, as @Makman2 said, adding a UnpickingError should suffice. The 
> worst that will happen in that case is that every project will need to be 
> cached again (maybe we can display an error saying caching broke, so every 
> project needs to be cached again). I don't think that's *too* bad.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/coala/pull/2016/files/621fc8d315bd5a587846946475b6ce9370489585#r59833966